Mathlib/Tactic/Find.lean

```diff
@@ -1,33 +1,102 @@
 /-
 Copyright (c) 2021 Sebastian Ullrich. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
-Authors: Sebastian Ullrich
+Authors: Sebastian Ullrich, Joachim Breitner
 -/
 import Lean
+import Lean.Data.Trie
+import Std.Lean.Delaborator
+import Std.Data.String.Basic
+import Std.Util.Pickle
 import Mathlib.Tactic.Cache
+import Mathlib.Lean.Data.NameRel
+import Mathlib.Lean.Data.RBTree
 
 /-!
 # The `#find` command and tactic.
```

Comment:
Can you reference the actual tactic in this docstring (by adding a `(name:=)` to the `#find` elab), so that it renders in doc-gen as a link, and this isn't a documentation regression?

I think it would be reasonable to mention loogle in this docstring too.